docs/container.md

```diff
@@ -27,44 +27,73 @@ The GPU enabled images are not currently tested by CI beyond being built. They a
 
 ## Usage
 
-The easiest way to download the models, convert them to ggml and optimize them is with the --all-in-one command which includes the full docker image.
+The easiest way to download the models, convert them to ggml and optimize them is with the --all-in-one command which includes the full container image.
 
 Replace `/path/to/models` below with the actual path where you downloaded the models.
 
 ```bash
 docker run -v /path/to/models:/models ghcr.io/ggerganov/llama.cpp:full --all-in-one "/models/" 7B
 ```
+or
```

Comment:
IMO this is now a bit hard to follow, because some people just copy-paste the command one by one without looking at the "or" word.

A better way here is to hide these podman code blocks inside a [expandable block](https://gist.github.com/scmx/eca72d44afee0113ceb0349dd54a84a2)